PYME/ui/traitsui_constants.py

```diff
@@ -43,7 +47,7 @@
 
 #: Color for background of windows (like dialog background color)
 if is_mac:
-    WindowColor = wx.Colour(232, 232, 232)
+    WindowColor = wx.SystemSettings.GetColour(wx.SYS_COLOUR_WINDOW)
```

Comment:
This looks too dark - maybe `SYS_COLOUR_MENUBAR` as on windows ??

Reply:
MENUBAR is a bit bright in my opinion:

<img width="324" alt="image" src="https://github.com/python-microscopy/python-microscopy/assets/1263313/63857918-a491-4920-8a88-e8e0ac75358b">

Reply:
`wx.SYS_COLOUR_INFOBK` looks good to me:

<img width="324" alt="image" src="https://github.com/python-microscopy/python-microscopy/assets/1263313/763514a8-3a90-49a1-a214-d2d752df4a05">

What do you think?

Reply:
Looks OK, but might be slightly safer as `wx.SYS_COLOUR_3DFACE` (which is what wx uses for panels - so the rest of the dialog). It appears to be numerically the same colour on mac, but who knows if it will be portable ... 

I'll change it.

Reply:
Sounds good. How do you know that `wx` uses `SYS_COLOUR_3DFACE` for panels? I've been looking at https://docs.wxpython.org/wx.SystemColour.enumeration.html to figure out these colorings. All I see there is that `SYS_COLOUR_3DFACE` is a synonym for `SYS_COLOUR_BTNFACE`, which is the face shading color on push buttons. Is there better documentation somewhere else?